src/dirs.proj

```diff
@@ -4,6 +4,7 @@
 
   <ItemGroup>
     <Project Include="*\**\*.csproj" />
+    <Project Include="*\**\*.depproj" />
```

Comment:
DepProj should only be used when you don't already have a project.json.  The built in nuget support is supposed to be all you need for deploying your own dependencies.